ntp_client.conf

```diff
@@ -0,0 +1,11 @@
+# In order to use this, move this file to /etc/ntp.conf (requires sudo)
```

Comment:
what's the diff btwn these two `ntp.conf` files and how are they used?

Reply:
I'll make a note in each file